### PR TITLE
Use pki_types::UnixTime instead of local Time type

### PR DESCRIPTION
**Pull request by djc**

### CODE REVIEW

File: src/time.rs [hunk 1]
```diff
@@ -106,6 +89,7 @@ impl<'a> FromDer<'a> for Time {
     const TYPE_ID: DerTypeId = DerTypeId::Time;
 }
 
+/*
```
Comment (cpu) [inline src/time.rs:92]: Why not remove this outright?

Comment (djc) [reply src/time.rs]: I wasn't sure yet if anything needed it. :)

**Review by cpu — APPROVED** [review]
LGTM :hourglass_flowing_sand:

**Review by ctz — APPROVED** [review]

**Comment by djc** [comment]
Published alpha.3.

**Pull request merged**